### ISSUE TITLE
Feature request: Popover overlay

### ISSUE THREAD

**Issue by tlrib** [huntabyte/shadcn-svelte#294]
### Describe the feature in detail (code, mocks, or screenshots encouraged)

It would be nice to have the option to add an overlay to the Popover component like done in, for example, headless UI (https://headlessui.com/react/popover#adding-an-overlay).

Maybe this falls under bits ui or meltui.

Thanks!

### What type of pull request would this be?

None

**Comment by huntabyte** [comment]
Yeah this would be a bits thing, so I'll transfer the issue over to that repo!
